docs/specs/values.md

```diff
@@ -31,6 +31,72 @@ RGB component.
 
 * {lottie_hexcolor: #FF8000}
 
+<h2 id="gradient">Gradient</h2>
+
+{schema_string:values/gradient/description}
+
+The count of color components is typically specified in a separate field from
+the gradient values. Any remaining values after reaching the specified count of
+color components are transparency components.
+
+The offset is expressed as a percentage between start (0) and end (1).
+
+Components must be arranged in ascending offset order. 
+
+<h3>Gradient without transparency</h3>
+
+So let's say you want these colors:
+
+* {lottie_color:0.16, 0.18, 0.46}
+* {lottie_color:0.2, 0.31, 0.69}
+* {lottie_color:0.77, 0.85, 0.96}
+
+the array will look like the following:
+
+`[0, 0.16, 0.18, 0.46, 0.5, 0.2, 0.31, 0.69, 1, 0.77, 0.85, 0.96]`
+
+| Value     | Description |
+|-----------|---|
+| `0`       | Offset of the 1st color (`0` means at the start) |
+| `0.16`   | Red component for the 1st color |
+| `0.18`   | Green component for the 1st color |
+| `0.46`   | Blue component for the 1st color |
+| `0.5`     | Offset of the 2nd color (`0.5` means half way) |
+| `0.2`   | Red component for the 2nd color |
+| `0.31`   | Green component for the 2nd color |
+| `0.69`    | Blue component for the 2nd color |
+| `1`       | Offset of the 3rd color (`1` means at the end) |
+| `0.77`   | Red component for the 3rd color |
+| `0.85`   | Green component for the 3rd color |
+| `0.96`   | Blue component for the 3rd color |
```

Comment:
Probably we want an embedded gradient preview. If needed I can add it later since I'm familiar with the markdown extension.

Reply:
There is an example/playground with the gradient with transparency. Did you want one for the non-transparency gradient as well or are you referring to something else with preview?

Reply:
I mean like the inline thing we have for colors, just to have a visual representation after the table

Reply:
Ah got it. I see the scripts for it, but couldn't get it to work. Is there an example somewhere? Or I could leave it to you.

Reply:
I can handle it, once all the current MRs are merged I can go through and make an editorial pass to make sure everything is consistent